src/context/body.test.ts

```diff
@@ -10,7 +10,7 @@ describe('body', () => {
     })
 
     it('should not have any "Content-Type" header set', () => {
-      expect(result.headers.get('Content-Type')).toEqual(null)
+      expect(result.headers.get('content-type')).toBeUndefined()
```

Comment:
This violates backwards compatibility with the `Headers` spec. This should return `null`.